python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -152,6 +152,30 @@ def run(self, patchRef, selectDataList=[]):
                 self.log.warn("tempExp %s could not be created", tempExpRef.dataId)
         return dataRefList
 
+    def _getCovMultiplier(self, calexpRefList, modelPsf, skyInfo):
+        multX = 0
+        multY = 0
+        for calExpInd, calExpRef in enumerate(calexpRefList):
+            try:
+                ccdId = calExpRef.get("ccdExposureId", immediate=True)
+            except Exception:
+                ccdId = calExpInd
+            calExpRef = calExpRef.butlerSubset.butler.dataRef("calexp", dataId=calExpRef.dataId,
+                                                              tract=skyInfo.tractInfo.getId())
+            calExp = self.getCalExp(calExpRef, bgSubtracted=self.config.bgSubtracted)
+            warpRes = self.warpAndPsfMatch.run(calExp, modelPsf=modelPsf, wcs=skyInfo.wcs,
+                                               maxBBox=skyInfo.bbox)
+            exposure = warpRes.exposure
+            covImage = warpRes.covImage
+            try:
+                multX = int(covImage.getWidth()/exposure.getWidth())
+                multY = int(covImage.getHeight()/exposure.getHeight())
+            except ZeroDivisionError:
+                continue
+            else:
+                break
+        return (multX, multY)
```

Comment:
Fixed - we now start with multX & multY  = 1 and break out of the loop the first time that we obtain any value for them.